packages/mwp-app-render/src/components/connectedWithMatchMedia.test.jsx

```diff
@@ -0,0 +1,38 @@
+import jsdom from 'jsdom';
+const doc = jsdom.jsdom('<!doctype html><html><body></body></html>');
+global.document = doc;
+global.window = doc.defaultView;
+
+import React from 'react';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import connectedWithMatchMedia from './connectedWithMatchMedia';
+
+export const createFakeStore = fakeData => ({
+	getState() {
+		return fakeData;
+	},
+	dispatch() {},
+	subscribe() {},
+});
+
+const TestComponent = () => (<div>Hello World</div>);
```

Comment:
Prettier should remove the redundant parens

Reply:
hmm prettier hasn't been running on commit for these new files